auth.proto

```diff
@@ -6,21 +6,26 @@ syntax = "proto3";
 package magistrala;
 option go_package = "./magistrala";
 
-// AuthzService is a service that provides authentication and authorization
+// ThingsService is a service that provides authentication and authorization
 // functionalities for the things service.
-service AuthzService {
+service ThingsService {
   // Authorize checks if the subject is authorized to perform
   // the action on the object.
   rpc Authorize(AuthorizeReq) returns (AuthorizeRes) {}
 }
 
 // AuthService is a service that provides authentication and authorization
-// functionalities for the users service.
+// functionalities for magistrala services.
 service AuthService {
   rpc Issue(IssueReq) returns (Token) {}
   rpc Refresh(RefreshReq) returns (Token) {}
   rpc Identify(IdentityReq) returns (IdentityRes) {}
   rpc Authorize(AuthorizeReq) returns (AuthorizeRes) {}
```

Comment:
Let's remove this method forom this Service. We already have it in AuthZService. We can, however, have a service (and the client) that implements both interfaces.

Reply:
Something like this:
![image](https://github.com/user-attachments/assets/798fdc9d-1707-4baa-992b-2b3d69a02a91)

Reply:
Exactly. 👍🏽